### ISSUE TITLE
No flushing for `println`, `@info` etch in notebooks

### ISSUE THREAD

**Issue by CarloLucibello** [julia-vscode/julia-vscode#2874]
When executing a cell like this 
```julia
for i in 1:5
    println("hi!")
    sleep(1)
end
```
each print is not flushed immediately but the output is displayed only at the end of the whole execution. 
Replacing `println` with `display` instead, I get a print every second as expected.

**Comment by CarloLucibello** [comment]
with `flush(stdout)` the `println` is flushed, but not `@info`.

**Comment by fredrikekre** [comment]
Logging goes to `stderr`, normally.

**Comment by PaulSoderlind** [comment]
Any updates on this? (I get random flushing when printing matrices in a notebook.)

**Comment by rbeeli** [comment]
Yes, please fix, IMO for each println, @ info, display, show etc. call, the cell output buffer should be flushed. Otherwise, the dev experience is pretty bad. Adding manual flush(stdout) is not a solution. This issue is a major blocker for recommending Julia and VSCode notebooks! Thank you.

**Comment by PaulSoderlind** [comment]
bump
